features/theme.feature

```diff
@@ -517,3 +517,30 @@ Feature: Manage WordPress themes
       """
       Error: Can't find the requested theme's version 1.4.2 in the WordPress.org theme repository (HTTP code 404).
       """
+
+  Scenario: Get status field in theme detail
+    Given a WP install
+
+    When I run `wp theme install p2`
+    Then STDOUT should not be empty
+
+    When I run `wp theme get p2`
+    Then STDOUT should contain:
```

Comment:
@sagarnasit Could you update this part to be something like https://github.com/wp-cli/extension-command/blob/08984ffe611894055ebcb21b3f6bd975184fa2fe/features/theme-mod.feature#L20-L22